app/helpers/application_helper/button/volume_backup_create.rb

```diff
@@ -0,0 +1,10 @@
+class ApplicationHelper::Button::VolumeBackupCreate < ApplicationHelper::Button::Basic
+  def calculate_properties
```

Comment:
@martinpovolny @tzumainn, the [`GenericFeatureButtonWithDisable`](https://github.com/tzumainn/manageiq/blob/a90845caf9424917da39dc95b3228e3ef3256a64/app/helpers/application_helper/button/generic_feature_button_with_disable.rb) is what you need :wink: